### ISSUE TITLE
Grunt dist: getting image path relative to css file

### ISSUE THREAD

**Issue by pasine** [yeoman/generator-webapp_DEPRECATED#229]
I want to keep my image background relative to css files, so I can move the application in any server subfolder without worries.
I have tried different configurations, but I had no success.
This is what I tried.

Compass options:

```
     compass: {
            options: {
                sassDir: '<%= yeoman.app %>/styles',
                cssDir: '.tmp/styles',
                generatedImagesDir: '.tmp/images/generated',
                imagesDir: '<%= yeoman.app %>/images',
                javascriptsDir: '<%= yeoman.app %>/scripts',
                fontsDir: '<%= yeoman.app %>/styles/fonts',
                importPath: '<%= yeoman.app %>/bower_components',
                httpImagesPath: '/images',
                httpGeneratedImagesPath: '/images/generated',
                httpFontsPath: '/styles/fonts',
                relativeAssets: false,
                assetCacheBuster: false
            },
            dist: {
                options: {
                    generatedImagesDir: '<%= yeoman.dist %>/images/generated'
                }
            },
            server: {
                options: {
                    debugInfo: true
                }
            }
        }
```

**First try**:
Trying to setup sprite images

```
main.scss
  @import "../images/sprite-*.png";
  @include all-images-sprites;

dist main.css:
  background:url(/images/generated/../cf762071.images-s4946e67892.png)
```

**Second try:**
Thought the problem was in sprite images, I used compass image-url

```
main.scss
  background: image-url('sprite-fff.png');

dist main.css:
  background:url(/images/e5be248d.sprite-fff.png);
```

**Third try:**
Trying to add the relativeAssets option in Gruntfile.js

```
Gruntfile.js
  relativeAssets: false

main.scss
  background: image-url('sprite-fff.png');

dist main.css:
  background: url('../../app/images/sprite-fff.png');
```

**The solution**
Using just plain css background property, the path is correct

```
Gruntfile.js
  relativeAssets: <true/false> // it's the same

main.scss
  background: url('../images/sprite-fff.png');

dist main.css:
  background: url('../images/sprite-fff.png');
```

Is there any configuration I can setup to use compass mixin and keep the relative path as result? I want to use them to generate sprite images.
I have tried to play with `httpImagesPath` and `httpGeneratedImagesPath` in `dist options`, but without luck.

**Comment by eddiemonge** [comment]
You should open this in github.com/yeoman/grunt-usemin

**Comment by albertpeiro** [comment]
Anyone facing problems with /dist css not picking up relatively called images I fixed this with the help of:
http://stackoverflow.com/questions/14316643/yeoman-css-image-paths

I arrived here after playing about with Yeoman for first time (but I already knew grunt and bower). I believe it's a common need to add images relatively called from css. Even more so when the tempting /images path is auto-created there after a nice yeoman tutorial. But I also understand yeoman users need to face "problems" and learn how to fix gruntfiles... so I'm cool with a no-fix of this... if that was even a goal for yeoman anyway.
